airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/build_image/steps/java_connectors.py

```diff
@@ -15,7 +15,7 @@ class BuildConnectorDistributionTar(GradleTask):
     A step to build a Java connector image using the distTar Gradle task.
     """
 
-    title = "Build connector tar"
+    title = "Build connector tar"  # type: ignore
```

Comment:
I'm generally cautious of inline type ignores — usually that's a sign our tooling is not setup for what we are trying to do. 

Is there a setting that we should set so that literals are better compatible? /cc @alafanechere